Tools/Scripts/libraries/resultsdbpy/resultsdbpy/__init__.py

```diff
@@ -24,6 +24,9 @@
 import platform
 import sys
 
+if sys.version_info < (3, 6):
+    raise ImportError("resultsdbpy requires Python 3.6 or above")
+
```

Comment:
We shouldn't need this, `resultsdbpy` has never supported Python 2 (and I think requires `3.7` or `3.8`?)

Reply:
To be fair, it doesn't seem harmful to have this here—IIRC, from memory, the tests all pass with Python 3.6, but we can of course bump this higher (but I was starting with being as conservative as possible!).